yarn.lock

```diff
@@ -27412,16 +27413,16 @@ __metadata:
   linkType: hard
 
 "graphql-language-service@npm:^5.0.6":
-  version: 5.0.6
-  resolution: "graphql-language-service@npm:5.0.6"
+  version: 5.1.7
+  resolution: "graphql-language-service@npm:5.1.7"
```

Comment:
A lot of upgrades in here seem unrelated. Maybe revert yarn.lock to its master shape and then run yarn install again?